src/GameStartup/GameStartupDriver.cpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include "GameStartup.h"
+#include "GameStartupDriver.h"
+#include "GameState.h"
+
+void gameStartupExample()
+{
+  GameState state();
+  std::cout << "When the game state is created, the deck is shuffled.\n";
+  std::cout << "At the same time 6 cards are drawn.\n";
+  std::cout << "In the two game states below, the cards are shuffled different.\n";
+  std::cout << "They are drawn from the top of shuffled deck and are therefore different.\n";
+  GameState state1 = GameState();
+  GameState state2 = GameState();
```

Comment:
What's the second state for?

Reply:
Testing purproses! To show that they have different draw piles.